share/ruby-build/truffleruby+graalvm-dev

```diff
@@ -1,26 +1,24 @@
 platform="$(uname -s)-$(uname -m)"
 case $platform in
 Linux-x86_64)
-  url="https://raw.githubusercontent.com/graalvm/graal-languages-ea-builds/main/truffleruby/versions/latest-jvm-linux-amd64.url"
+  url="https://github.com/graalvm/graal-languages-ea-builds/raw/HEAD/truffleruby/versions/latest-jvm-linux-amd64.url"
```

Comment:
Let's stick to using `github.com` URLs instead of `raw.githubusercontent.com` since the latter is an implementation detail of GitHub which might change in the future.